src/background_jobs.c

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
This should be bgw_db_launcher.c or somesuch. It should only contain stuff about the db launcher and nor background workers in general.